docs/feature/002-vip-counter-1.md

```
@@ -0,0 +1,38 @@
+> ipv4     2 udp      17 14 src=192.168.100.254 dst=192.168.100.147 sport=43329 dport=53 packets=2 bytes=118 [UNREPLIED] src=10.1.2.197 dst=192.168.100.254 sport=53 dport=43329 packets=0 bytes=0 mark=0 zone=0 use=2
+
+1.4 其它格式类似, 每个一行包含了出入两个方向, 
+
+1.5 把前述数据转化成一个go map, key是前一半的五元组，value是这个五元组的流量统计packets,bytes 
+
+
+2. 完成当前的session的流量统计后，遍历每个session:
+2.1 vip.go 中vipAddrInfoMap记录vip地址和uuid的映射关系
+2.2 vip.go 中定义了vip的流量统计格式:
+> type VipCounter struct {
+>	Packets      uint64
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 2

(MD010, no-hard-tabs)